h/util/logging_filters.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""Logging Filters."""
+
+import logging
+
+
+LEVELS = {"CRITICAL": 50,
```

Comment:
Perhaps worth adding a brief comment that these come from https://docs.python.org/2/library/logging.html#logging-levels .